README.md

```diff
@@ -3,6 +3,55 @@
 A [Singer](https://singer.io) Tap to extract currency exchange rate
 data from [fixer.io](http://fixer.io).
 
+## How to use it
+
+`tap-fixerio` works to pull currency exchange rates from the Fixer API
```

Comment:
This seems redundant with lines 3 and 4.

Reply:
agreed, I felt weird even putting it. I was trying to be consistent with other READMEs but really my focus should have been on valuable written communication instead of copying and pasting.